dashboard/app/models/blockly.rb

```diff
@@ -25,6 +25,7 @@
 require 'nokogiri'
 class Blockly < Level
   include SolutionBlocks
+  include LocaleHelper
```

Comment:
It very likely should be, but given that I'm only including it here for `data_t` I will for this purpose take your suggestion and just call `I18n.t`